src/components/com_tjcertificate/site/models/externalcertificate.php

```diff
@@ -0,0 +1,261 @@
+<?php
```

Comment:
can you  move this new externalcertificate controller, model to admin side like other controller and models

Reply:
done